include/llbuild/Basic/CrossPlatformCompatibility.h

```diff
@@ -18,16 +18,47 @@
 #define CrossPlatformCompatibility_h
 
 #ifdef _MSC_VER
```

Comment:
Assuming you're using MSVC and not Clang for testing thing, I'd just like to note for the record that Clang defines both _MSC_VER and _WIN32 for compatibility with MSVC when targeting Windows, so this branch actually does mean Windows in general, not *only* Windows+MSVC.

If you definitely need something to be MSVC specific and NOT apply to Clang, you'll need to use something like `#if (defined(_WIN32) || defined(_MSC_VER)) && !defined(__clang__)`